addons/account/tests/test_reconciliation.py

```diff
@@ -2131,3 +2126,275 @@ def test_reconciliation_with_currency(self):
         to_reconcile.reconcile()
         for aml in to_reconcile:
             self.assertEqual(aml.amount_residual, 0.0)
+
+
+@tagged('post_install', '-at_install')
+class TestReconciliationInvoiceWidgets(TestReconciliation):
```

Comment:
I'm trying to understand your tests but it's really unclear. Why not making something you could compute yourself using only your brain? For example, make some invoices having 1200, 2400, 4800 amounts and set your currency rates at 2, 3, 4, 6 ?

Reply:
lol, sorry
But amounts and exchange rate matter to account for rounding or conversion issues

Reply:
@smetl by using our brains is why we came up here.